cmake/std/PullRequestLinuxPython2.cmake

```diff
@@ -14,7 +14,26 @@
 set (TFW_Python2_Testing ON CACHE BOOL "Set by default for PR testing")
 set (TFW_Python_Testing ON CACHE BOOL "Set by default for PR testing")
 
-set(PYTHON_EXECUTABLE /projects/sierra/linux_rh7/install/Python/2.7.15/bin/python CACHE FILEPATH "Set by default for PR testing")
+#set(PYTHON_EXECUTABLE /projects/sierra/linux_rh7/install/Python/2.7.15/bin/python CACHE FILEPATH "Set by default for PR testing")
+#set(PYTHON_EXECUTABLE /projects/sierra/linux_rh7/install/Python/2.7.5/bin/python CACHE FILEPATH "Set by default for PR testing")
+set(PYTHON_PIP_EXECUTABLE "pip2" CACHE STRING "Set by default for PR testing")
+
+
+set(PYTHON_EXECUTABLE_SEARCH_PATHS
+    /projects/sierra/linux_rh7/install/Python/2.7.15
+    /projects/sierra/linux_rh7/install/Python/2.7.5
```

Comment:
Long story - basically because testing using the sems installs is difficult (no mock package.) We are moving towards python3 only which will help remove this issue. Keeping both complicates a few other things as well.

Reply:
This ^

Also, if SEMS would get around to taking care of that request I filed a long time ago and get us a Python >= 3.6 it'd be super nice.  I think they had some reason why they were going to put that off but I don't recall what it was, but not having access to anything 3.6 or greater means that some python modules will fail to install if they are using f-strings. :/ 

Feel free to file additional tickets against them with the request :p